packages/engine/paima-funnel/src/funnels/emulated/utils.ts

```diff
@@ -54,8 +54,8 @@ export function emulateCde(
     .flat()
     .sort((a, b) => {
       const bhDiff = a.blockNumber - b.blockNumber;
-      const cdeIdDiff = a.cdeId - b.cdeId;
-      return bhDiff ? bhDiff : cdeIdDiff;
+      const cdeNameDiff = a.cdeName.localeCompare(b.cdeName);
```

Comment:
this is a breaking change we'll have to mention

Reply:
mention where though? are we keeping a changelog?

Reply:
yeah I write the changelog when I release new versions. This comment is mostly for me to remember